lib/movable_ink/aws/ec2.rb

```diff
@@ -244,6 +244,23 @@ def redis(host, port)
         redis_instances.flatten.shuffle
       end
 
+      def describe_ip(public_ip:)
+        expected_errors = [
+          MovableInk::AWS::Errors::ExpectedError.new(Aws::EC2::Errors::InvalidAddressNotFound, [/Address \'#{public_ip}\' not found./])
+        ]
+        begin
+          run_with_backoff(expected_errors: expected_errors) do
+            response = ec2.describe_addresses({
```

Comment:
Can this use `ec2_with_retries`, so that it automatically retries getting the credentials?